pkg/question/question.go

```diff
@@ -226,35 +225,40 @@ func AskLaunchTemplate(h *ec2helper.EC2Helper, defaultLaunchTemplateId string) *
 	indexedOptions := []string{}
 
 	noUseOptionRepr, noUseOptionValue := "Do not use launch template", cli.ResponseNo
-	defaultOptionRepr, defaultOptionValue := noUseOptionRepr, noUseOptionValue
+	defaultOption := noUseOptionValue
 	// Fill the data used for drawing a table and the options map
-	for index, launchTemplate := range launchTemplates {
+	for _, launchTemplate := range launchTemplates {
```

Comment:
I'd prefer to keep this PR as focused as possible. We have three actual question implementations now, so let's save any more for a separate PR.